frontend/src/got/characterCard/CharacterCardGoT.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {GoTCharacter} from "../model/GoTCharacter";
+
+type Props={
+    character:GoTCharacter
```

Comment:
Sind props mit einer Eigenschaft sinnvoll? Alternativ kann man hier auch direkt nur den character übergeben

Reply:
is voll sinvoll, erweiterbarkeit ist vor allem gegeben.